src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -329,6 +334,8 @@ Package url{6}
                                     package.Summary != null && !string.IsNullOrWhiteSpace(package.Summary.ToStringSafe()) ? "\r\n Summary: {0}".FormatWith(package.Summary.EscapeCurlyBraces().ToStringSafe()) : string.Empty,
                                     package.Description.EscapeCurlyBraces().Replace("\n    ", "\n").Replace("\n", "\n  "),
                                     !string.IsNullOrWhiteSpace(package.ReleaseNotes.ToStringSafe()) ? "{0} Release Notes: {1}".FormatWith(Environment.NewLine, package.ReleaseNotes.EscapeCurlyBraces().Replace("\n    ", "\n").Replace("\n", "\n  ")) : string.Empty,
+                                    !string.IsNullOrWhiteSpace(deploymentlocation) ? "{0} Deployed to: '{1}'".FormatWith(Environment.NewLine, deploymentlocation) :string.Empty,
+                                    !string.IsNullOrWhiteSpace(packageInstalledFrom) ? "{0} Package Installed From: '{1}'".FormatWith(Environment.NewLine, packageInstalledFrom) : string.Empty,

```

Comment:
```suggestion
                                    !string.IsNullOrWhiteSpace(packageInstalledFrom) ? "{0} Package installed from: '{1}'".FormatWith(Environment.NewLine, packageInstalledFrom) : string.Empty,
```

I don't "know" is this is right or not.  Perhaps a final check from @pauby here?

Reply:
Spoke with Gary about this, he and I were looking at different parts of this suggestion. We've settled on the wording "Source package was installed from" as it matches the name of the variable and is more clear what it's referring to.